docs/diagrams/DeleteFieldSequenceDiagram.puml

```diff
@@ -0,0 +1,90 @@
+@startuml
```

Comment:
Are the following parts considered as mistakes? (Refer to image for the numbering)

1. The activation bar should be connected to the class name?
2. Should `p` be returned from the constructor?
3. Should `c` be returned from the constructor?
4. Are there missing dotted arrows to denote the return from `model`?
5. Should `cr` be returned from `LogicManager`?


![image](https://user-images.githubusercontent.com/86542359/230147036-9ec1aecd-30f4-4b07-99f2-1dbb21e256cd.png)

Reply:
Thanks for the review. I have a problem with point 1. If I split the constructor into 2 lines (`internshipContainsKeywordPredicate(...)`), there will be a gap. I had to add a new line to the blue box. Is it ok?;

![image](https://user-images.githubusercontent.com/105597933/230464228-1d5ade0a-7868-4bbd-8cfc-0ad7a2c40e07.png)

Reply:
Yes, it looks great! 👍